src/components/game/game.jsx

```diff
@@ -63,60 +26,50 @@ class Game extends Component {
 
     let status;
     if (winner) {
```

Comment:
also can we put this logic in a a function? i.e. `const status = calculateWinnerStatus(winner)`

Reply:
Done